web/pgadmin/misc/cloud/google/tests/test_google_oauth2client_blocked.py

```
@@ -0,0 +1,58 @@
+class _SkipServerSetUpMixin:
+    """Skip BaseTestGenerator's Postgres connection — pure logic tests."""
+
+    def setUp(self):
+        unittest.TestCase.setUp(self)
+
+
+class TestOauth2ClientBlockedOnGoogleImport(
+        _SkipServerSetUpMixin, BaseTestGenerator):
+    """Importing cloud.google must keep googleapiclient off oauth2client."""
+
+    scenarios = [('default', dict())]
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)